api/controller.py

```diff
@@ -714,8 +714,8 @@ def groups(self, lane_identifier, feed_class=AcquisitionFeed):
         if isinstance(lane, ProblemDetail):
             return lane
 
-        if not lane.sublanes:
-            # This lane has no sublanes. Although we can technically
+        if not lane.children:
+            # This lane has no children. Although we can technically
```

Comment:
I could not figure out how to test this, but this change crashes the main web page in a real application.